gpos/xml/scap/ssg/content/ssg-chainguard-xccdf/OvalChecks/XccdfOvalChecks.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" ?>
-<Benchmark xmlns="http://checklists.nist.gov/xccdf/1.2" xmlns:html="http://www.w3.org/1999/xhtml" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="xccdf_oval_benchmark_.beta" style="SCAP_1.2" resolved="false" xml:lang="en-US" xsi:schemaLocation="http://checklists.nist.gov/xccdf/1.2 xccdf-1.2.xsd">
+<Benchmark xmlns="http://checklists.nist.gov/xccdf/1.2" xmlns:html="http://www.w3.org/1999/xhtml" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:cpe2="http://cpe.mitre.org/language/2.0" id="xccdf_oval_benchmark_.beta" style="SCAP_1.2" resolved="false" xml:lang="en-US" xsi:schemaLocation="http://checklists.nist.gov/xccdf/1.2 xccdf-1.2.xsd http://cpe.mitre.org/language/2.0 cpe-language_2.0.xsd">
   <status date="2023-04-01">draft</status>
   <title>XCCDF Checklist Beta</title>
   <description>This XCCDF document provides a basic security check for a Linux system.</description>
   <notice id="legal">Copyright (c) 2023, Your Organization. All rights reserved.</notice>
   <reference href="http://yourorganization.com/security-policies">Security Policy Reference</reference>
+  <platform idref="cpe:/o:linux:linux_kernel"/>
```

Comment:
not sure this is correct, especially since our containers are not shipping a kernel.

Reply:
Fixed in: [`35c8b42` (#20)](https://github.com/chainguard-dev/stigs/pull/20/commits/35c8b42d08dabf4f04c912a8a698ee9fd7a423d6)